raspiraw.c

```diff
@@ -216,57 +218,75 @@ enum {
 
-static COMMAND_LIST cmdline_commands[] =
-{
-	{ CommandHelp,		"-help",	"?",  "This help information", 0 },
-	{ CommandMode,		"-mode",	"md", "Set sensor mode <mode>", 1 },
-	{ CommandHFlip,		"-hflip",	"hf", "Set horizontal flip", 0},
-	{ CommandVFlip,		"-vflip",	"vf", "Set vertical flip", 0},
-	{ CommandExposure,	"-ss",		"e",  "Set the sensor exposure time (not calibrated units)", 0 },
-	{ CommandGain,		"-gain",	"g",  "Set the sensor gain code (not calibrated units)", 0 },
-	{ CommandOutput,	"-output",	"o",  "Set the output filename", 0 },
-	{ CommandWriteHeader,	"-header",	"hd", "Write the BRCM header to the output file", 0 },
-	{ CommandTimeout,	"-timeout",	"t",  "Time (in ms) before shutting down (if not specified, set to 5s)", 1 },
-	{ CommandSaveRate, 	"-saverate",	"sr", "Save every Nth frame", 1 },
-	{ CommandBitDepth, 	"-bitdepth",	"b",  "Set output raw bit depth (8, 10, 12 or 16, if not specified, set to sensor native)", 1 },
-	{ CommandCameraNum, 	"-cameranum",	"c",  "Set camera number to use (0=CAM0, 1=CAM1).", 1 },
-	{ CommandExposureus, 	"-expus",	"eus",  "Set the sensor exposure time in micro seconds.", -1 },
-	{ CommandI2cBus, 	"-i2c",	        "y",  "Set the I2C bus to use.", -1 },
-	{ CommandAwbGains, 	"-awbgains",	"awbg", "Set the AWB gains to use.", 1 },
-	{ CommandRegs,	 	"-regs",	"r",  "Change (current mode) regs", 0 },
-	{ CommandHinc,		"-hinc",	"hi", "Set horizontal odd/even inc reg", -1},
-	{ CommandVinc,		"-vinc",	"vi", "Set vertical odd/even inc reg", -1},
-	{ CommandFps,		"-fps",		"f",  "Set framerate regs", -1},
-	{ CommandWidth,		"-width",	"w",  "Set current mode width", -1},
-	{ CommandHeight,	"-height",	"h",  "Set current mode height", -1},
-	{ CommandLeft,		"-left",	"lt", "Set current mode left", -1},
-	{ CommandTop,		"-top",		"tp", "Set current mode top", -1},
-	{ CommandWriteHeader0,	"-header0",	"hd0","Sets filename to write the BRCM header to", 0 },
-	{ CommandWriteHeaderG,	"-headerg",	"hdg","Sets filename to write the .pgm header to", 0 },
-	{ CommandWriteTimestamps,"-tstamps",	"ts", "Sets filename to write timestamps to", 0 },
-	{ CommandWriteEmpty,	"-empty",	"emp","Write empty output files", 0 },
-	{ CommandDecodeMetadata,"-metadata",	"m","Decode register metadata", 0 },
-	{ CommandAwb,		"-awb",		"awb","Use a simple grey-world AWB algorithm", 0 },
-	{ CommandNoPreview,	"-nopreview",	"n",  "Do not send the stream to the display", 0 },
-	{ CommandProcessing,	"-processing",	"P",  "Pass images into an image processing function", 0 },
-	{ CommandPreview,	"-preview",	"p",  "Preview window settings <'x,y,w,h'>", 1 },
-	{ CommandFullScreen,	"-fullscreen",	"fs", "Fullscreen preview mode", 0 },
-	{ CommandOpacity,	"-opacity",	"op", "Preview window opacity (0-255)", 1},
-	{ CommandProcessingYUV,	"-processing_yuv", "PY",  "Pass processed YUV images into an image processing function", 0 },
-	{ CommandOutputYUV,	"-output_yuv",  "oY", "Set the output filename for YUV data", 0 },
+static COMMAND_LIST cmdline_commands[] = {
```

Comment:
Losing the tabulation of entries here makes it less readable.

And the space after {, and before } also help make it readable.

Reply:
Fixed.